foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -1354,21 +1354,19 @@ def cleanup_the_queue():
                 # will be incremented when it is requeued).
                 try:
                     nomad_client.job.deregister_job(job["ID"], purge=True)
-                    processor_jobs = ProcessorJob.objects.filter(nomad_job_id=job["ID"])
+                    logger.info('Foreman Killed nomad job because it did not have a volume assigned',
```

Comment:
Can you make this DEBUG? This could get noisy.

Reply:
Also the log statement isn't quite right. The job did have a volume assigned but the volume it was assigned to wasn't active.